graphyte.py

```diff
@@ -25,6 +25,12 @@
 default_sender = None
 logger = logging.getLogger(__name__)
 
+def has_whitespace(value):
```

Comment:
Can we change this to `_has_whitespace` to indicate it's private?

Reply:
Sure! Done